doc/nrf/ug_matter_configuring_env.rst

```diff
@@ -31,12 +31,54 @@ In this setup, Raspberry Pi runs the Thread Border Router, which provides commun
 The controller can be installed on a PC or a mobile phone.
 Both the Thread Border Router and the Matter controller must support IPv6 communication over backbone network, for example Wi-Fi or Ethernet.
 
+.. _ug_matter_configuring_pc_chip_tool:
+
+Configuring Thread Border Router and the CHIP Tool Matter controller on PC
```

Comment:
Pity that we have the tool named CHIP tool but the CHIP is the deprecated denotation of Matter... not sure we can refine this anyway :)